tests/test_feedback.py

```
@@ -0,0 +1,521 @@
+                "/api/feedback",
+                json={
+                    "feedback_type": "bug",
+                    "description": "Something broke",
+                },
+            )
+            assert resp.status_code == 503
+            assert "disabled" in resp.json()["detail"]
+
+    def test_successful_feedback_submission(self, _init_db, temp_db_path):
+        for client in self._make_client(
+            temp_db_path, github_token="ghp_test"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "github_token"

(S106)